vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -50,9 +55,9 @@ def forward(
         sequence.
 
         Args:
-            target_probs: The probability distribution over token ids given
-                context according to the target model.
-            shape = [batch_size, num_speculative_tokens, vocab_size]
+            target_probs_with_bonus_probs: The probability distribution 
+                over token ids given context according to the target model.
```

Comment:
should update the variable name `target_probs_with_bonus_probs ` here too https://github.com/vllm-project/vllm/blob/3979f69dbd8b36aa0286d09ac839c9cdc43a7f14/vllm/model_executor/layers/spec_decode_base_sampler.py#L215-L245